psi4/src/psi4/libmints/integral.cc

```diff
@@ -84,7 +82,7 @@ void IntegralFactory::set_basis(std::shared_ptr<BasisSet> bs1, std::shared_ptr<B
     bs4_ = bs4;
 
     // Use the max am from libint
-    init_spherical_harmonics(LIBINT_MAX_AM + 1);
+    init_spherical_harmonics(7);
```

Comment:
Does libint2 provide a macro for the max am?

Reply:
I think the difficulty is that l2 isn't loaded at this point. I did reference this in https://github.com/psi4/psi4/pull/1721/files#diff-f35a0bcb0ee227178fc9064649dd02291985af93a5995f2b4bb08e507e0d8985R24. We could probably boost this a couple more for safety (after all, it would have been 9, I think, for l1), but 12 shows performance problems.

Reply:
I remember that now.  Yeah, maybe bump it up ... but once everything is switched to libint2 I'm not sure this will even be needed any more.

Reply:
I'm pretty sure that's a compile-time option declared in the libint2 headers?

Reply:
I've changed it to 8. 9 sometimes added 20s to the total time of a tu1 * 50. I think we tried to pull it in directly from l2 once , but there were problems. Sounds like this may be unnecessary soon.